include/Zoo.h

```diff
@@ -0,0 +1,40 @@
+/* This is a header file for the zoo class that runs the game. */
+#include <vector>
+#include "Animal.h"
+#include <string>
+
+class Zoo {
+
+public:
+	// Constructor
+	Zoo();
+	void run();
+
+private:
+	// A list that will contain the animal instances
+	std::vector<Animal*> animalList;
+	
+	char map[20][40] = {};
+	
+	// The initial list
+	std::vector<std::vector<std::string>> lst{
```

Comment:
why you are using this list ???
you can get these info from the animal class

Reply:
I fixed it.